### PR TITLE
Chat 82 faq page

### PR DESCRIPTION
**Pull request by KyrychenkoDmytro**

### CODE REVIEW

**Review by DStukalo — CHANGES_REQUESTED** [review]
Accordeon don't change icon when he open. Accordeon icons don't change color in dark theme.

File: src/components/Layouts/Header/Header.jsx
Comment (Skynet4241) [inline src/components/Layouts/Header/Header.jsx:1]: Remove changes from Heder

File: src/App.jsx [hunk 1]
```diff
@@ -11,6 +12,7 @@ function App() {
         <Routes>
           <Route path="/" element={<SharedLayout />}>
             <Route index />
+            <Route path="/faq" element={<FaqPage />} />
```
Comment (Skynet4241) [inline src/App.jsx:15]: Delete it

File: src/components/ExpandIcon/ExpandIcon.jsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import { Box } from '@mui/material';
+import { OpenIcon, CloseIcon } from './ExpandIcon.styled';
+
+const ExpandIcon = () => {
+  return (
+    <Box
+      sx={{
```
Comment (Skynet4241) [inline src/components/ExpandIcon/ExpandIcon.jsx:7]: Delete the styles here and move them to file ExpandIcon.styled.js

File: src/components/ExpandIcon/ExpandIcon.jsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        },
+        '.expandIconWrapper': {
+          display: 'none',
+        },
+        '.Mui-expanded & > .expandIconWrapper': {
+          display: 'block',
+        },
+      }}
+    >
+      <div className="expandIconWrapper">
```
Comment (Skynet4241) [inline src/components/ExpandIcon/ExpandIcon.jsx:19]: Use Box from Material-UI / we write all styles through styled-components

File: src/components/ExpandIcon/ExpandIcon.jsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+        <CloseIcon />
+      </div>
+      <div className="collapsIconWrapper">
```
Comment (Skynet4241) [inline src/components/ExpandIcon/ExpandIcon.jsx:22]: And here too

File: src/pages/FaqPage/FaqPage.jsx [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+];
+
+const FaqPage = () => {
+  return (
+    <FaqPageWrapper>
+      <FaqPageTitle variant="h21">The most common questions</FaqPageTitle>
+      {titlesAndTexts.map(({ title, text }, index) => (
+        <FaqAccordion
+          key={index}
+          // sx={{
```
Comment (Skynet4241) [inline src/pages/FaqPage/FaqPage.jsx:38]: Delete it

File: src/ui-kit/theme/theme.js [hunk 6]
```diff
@@ -112,6 +112,7 @@ export const palette = (mode) => ({
     ...(mode === 'light'
       ? {
           primary: {
+            whiteRBGA: '#FFFFFF',
```
Comment (Skynet4241) [inline src/ui-kit/theme/theme.js:115]: What is this for?

File: src/ui-kit/theme/theme.js [hunk 7]
```diff
@@ -122,10 +123,15 @@ export const palette = (mode) => ({
             disabled: '#EEEFFF',
             errorColor: '#DA4444',
             lightDisabled: '#999999',
+            darkPurple: 'FFFFFF',
```
Comment (Skynet4241) [inline src/ui-kit/theme/theme.js:126]: What is this for?

File: src/pages/FaqPage/FaqPage.jsx [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+    title:
+      'Lorem ipsum dolor sii in ex eleifend, faucibus libero ac, lobortis leo.',
+    text: 'Lorem ipsum dolor sit amet, conseckskdfksdf - sasdfsdsdfdsf',
+  },
+];
+
+const FaqPage = () => {
+  return (
+    <FaqPageWrapper>
+      <FaqPageTitle variant="h21">The most common questions</FaqPageTitle>
```
Comment (Skynet4241) [inline src/pages/FaqPage/FaqPage.jsx:34]: Does not match the.

File: src/pages/FaqPage/FaqPage.jsx [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+        <FaqAccordion
+          key={index}
+          // sx={{
+          //   '.Mui-expanded': {
+          //     color: 'red',
+          //   },
+          // }}
+        >
+          <AccordionSummary expandIcon={<ExpandIcon />}>
+            {title}
```
Comment (Skynet4241) [inline src/pages/FaqPage/FaqPage.jsx:45]: The color does not match the layout

File: src/ui-kit/theme/theme.js [hunk 10]
```diff
@@ -122,10 +123,15 @@ export const palette = (mode) => ({
...
+            lightPurple: '6261AF',
```
Comment (Skynet4241) [inline src/ui-kit/theme/theme.js:127]: What is this for? Should be preceded by #

File: src/ui-kit/theme/theme.js [hunk 11]
```diff
@@ -135,7 +141,9 @@ export const palette = (mode) => ({
             contrastText: '#8686DC',
             disabled: '#353535',
             errorColor: '#DA4444',
+            darkPurple: '#4E4D87',
             lightDisabled: '#999999',
+            lightPurple: 'EEEFFF',
```
Comment (Skynet4241) [inline src/ui-kit/theme/theme.js:146]: What is this for? Should be preceded by #

**Review by Skynet4241 — APPROVED** [review]

**Review by DStukalo — APPROVED** [review]

**Pull request merged**